templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
```

Comment:
You can do that without the div or use a div with a custom class, but in that case I'd suggest a different name (e.g. "bordered-block" would work) to avoid confusion - we don't usually use class names that are the same as the names of existing HTML tags :)